docs/macbuild.rst

```diff
@@ -14,22 +14,25 @@ distribution to other systems.
    ``Allow apps downloaded from Anywhere``. This is necessary because
    GateKeeper will try to prevent this installation. You can return this
    setting to its previous state following the installation.
-2. Install the Python 2.7 Installer from the `Python download site`_
+2. Install the Python 2.7+ Installer from the `Python download site`_
```

Comment:
I think we should push Python 3 a little bit here. How about "Install the Python 3 Installer (or Python 2 if you have a specific reason for doing so) from the..."